libs/sdk-ui-filters/src/AttributeFilter@next/AttributeFilter.tsx

```diff
@@ -1,326 +1,27 @@
-     *
-     * @remarks
-     * Parent filters elements must contain their URIs due to current backend limitations.
-     */
-    parentFilters?: AttributeFiltersOrPlaceholders;
-
-    /**
-     * Specify {@link @gooddata/sdk-ui#IPlaceholder} to use to get and set the value of the attribute filter.
-     *
-     * @remarks
-     * Note: It's not possible to combine this property with "filter" property. Either - provide a value, or a placeholder.
-     * There is no need to specify 'onApply' callback if 'connectToPlaceholder' property is used as the value of the filter
-     * is set via this placeholder.
-     */
-    connectToPlaceholder?: IPlaceholder<IAttributeFilter>;
-
-    /**
-     * Specify the over attribute - an attribute the filter and its parent filter are connected through.
-     *
-     * @remarks
-     * You can either provide an {@link @gooddata/sdk-model#ObjRef} which will be used for all the parent filters,
-     * or you can provide a function that will be called for each parent filter to determine the respective over attribute.
-     */
-    parentFilterOverAttribute?: ObjRef | ((parentFilter: IAttributeFilter, index: number) => ObjRef);
-
-    /**
-     * Specify function which will be called when user clicks 'Apply' button.
-     *
-     * @remarks
-     * The function will receive the current specification of the filter, as it was updated by the user.
-     *
-     * @param filter - new value of the filter.
-     */
-    onApply?: (filter: IAttributeFilter) => void;
-
-    /**
-     * Specify title for the attribute filter.
-     *
-     * @remarks
-     * By default, the attribute name will be used.
-     */
-    title?: string;
-
-    /**
-     * Customize whether selected items should be summarized in the subtitle of the filter.
-     *
-     * @remarks
-     * Subtitle shows the list of selected elements divided by commas.
-     */
+export interface IAttributeFilterProps extends IAttributeFilterBaseProps {
     titleWithSelection?: boolean;
-
-    /**
-     * Customize, whether the filter should take the entire screen on mobile devices.
-     */
-    fullscreenOnMobile?: boolean;
-
-    /**
-     * Customize locale to use for the different strings that appear on the filter component.
-     */
-    locale?: string;
-
-    /**
-     * Customize attribute filter with a callback function to trigger when an error occurs while
-     * loading attribute elements.
-     */
-    onError?: OnError;
-
-    /**
-     * Customize attribute filter with a component to be rendered if attribute elements are loading
-     */
-    FilterLoading?: React.ComponentType;
-
-    /**
-     * Customize attribute filter with a component to be rendered if attribute elements loading fails
-     */
-    FilterError?: React.ComponentType<{ error?: any }>;
+    fullscreenOnMobile?: boolean; //not handled
+    FilterLoading?: React.ComponentType; //TODO it will be part of IAttributeFilterBaseProps
```

Comment:
Should we unify it with LoadingComponent? (`React.ComponentType<ILoadingProps>`)

Reply:
LoadingComponent will be added to IAttributeFilterBaseProps and it will be customisable